### PR TITLE
Alena burak/work questions 4

### PR DESCRIPTION
**Pull request by ElenaBurak**

### CODE REVIEW

**Review by romanstat — APPROVED** [review]

**Review by artemgael96 — APPROVED** [review]

File: exalearn-api/ExaLearn.Bl/Interfaces/ITestService.cs [hunk 1]
```diff
@@ -6,10 +6,10 @@ namespace ExaLearn.Bl.Interfaces
 {
     public interface ITestService
     {
-        Task<PassedTestForCheckDTO> GetUserTestByPassedTestIdAsync(int passedTestId, string checker);
+        Task<PassedTestForCheckDTO> GetUserTestByPassedTestIdAsync(int passedTestId, string checkerEmail);
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.Bl/Interfaces/ITestService.cs:10]: rename

File: exalearn-api/ExaLearn.Bl/Interfaces/ITestService.cs [hunk 2]
```diff
@@ -6,10 +6,10 @@ namespace ExaLearn.Bl.Interfaces
...
 
         Task<AssessmentDTO> CreateAssesmentAsync(AssessmentDTO assessmentDTO);
 
-        Task<IList<PassedTestDTO>> GetUnverifiedTestsForCoachAsync();
+        Task<IList<PassedTestDTO>> GetUnverifiedTestsForCoachAsync(string checkerEmail);
     }
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.Bl/Interfaces/ITestService.cs:14]: rename

File: exalearn-api/ExaLearn.Bl/Interfaces/ITestService.cs [hunk 3]
```diff
@@ -6,10 +6,10 @@ namespace ExaLearn.Bl.Interfaces
 {
     public interface ITestService
     {
-        Task<PassedTestForCheckDTO> GetUserTestByPassedTestIdAsync(int passedTestId, string checker);
+        Task<PassedTestForCheckDTO> GetUserTestByPassedTestIdAsync(int passedTestId, string checkerEmail);
 
         Task<AssessmentDTO> CreateAssesmentAsync(AssessmentDTO assessmentDTO);
 
-        Task<IList<PassedTestDTO>> GetUnverifiedTestsForCoachAsync();
+        Task<IList<PassedTestDTO>> GetUnverifiedTestsForCoachAsync(string checkerEmail);
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.Bl/Interfaces/ITestService.cs]: rename

File: exalearn-api/ExaLearn.Bl/Services/TestService.cs [hunk 4]
```diff
@@ -87,9 +87,10 @@ public async Task<AssessmentDTO> CreateAssesmentAsync(AssessmentDTO assessmentDT
             return _mapper.Map<AssessmentDTO>(assessment);
         }
 
-        public async Task<IList<PassedTestDTO>> GetUnverifiedTestsForCoachAsync()
+        public async Task<IList<PassedTestDTO>> GetUnverifiedTestsForCoachAsync(string checkerEmail)
         {
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.Bl/Services/TestService.cs:91]: rename

File: exalearn-api/ExaLearn.Bl/Services/TestService.cs [hunk 5]
```diff
@@ -87,9 +87,10 @@ public async Task<AssessmentDTO> CreateAssesmentAsync(AssessmentDTO assessmentDT
             return _mapper.Map<AssessmentDTO>(assessment);
         }
 
-        public async Task<IList<PassedTestDTO>> GetUnverifiedTestsForCoachAsync()
+        public async Task<IList<PassedTestDTO>> GetUnverifiedTestsForCoachAsync(string checkerEmail)
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.Bl/Services/TestService.cs]: rename

File: exalearn-api/ExaLearn.Dal/Entities/Question.cs [hunk 6]
```diff
@@ -13,10 +13,12 @@ public class Question
...
 
-        public string Text { get; set; }       
+        public string Text { get; set; }
 
         public List<Answer> Answers { get; set; } = new List<Answer>();
 
         public List<UserTest> UserTest { get; set; } = new List<UserTest>();
+
+        public bool? Archived { get; set; }
     }
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.Dal/Entities/Question.cs:23]: make isArchive

File: exalearn-api/ExaLearn.Dal/Interfaces/IPassedTestRepository.cs [hunk 7]
```diff
@@ -12,7 +12,7 @@ public interface IPassedTestRepository : IGenericRepository<PassedTest>
 
         Task<IList<PassedTest>> MyTestHistoryAsync(int userId);
 
-        Task<IList<PassedTest>> GetUnverifiedTestsAsync();
+        Task<IList<PassedTest>> GetUnverifiedTestsAsync(int checkerId);
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.Dal/Interfaces/IPassedTestRepository.cs:16]: rename to id

File: exalearn-api/ExaLearn.Dal/Interfaces/IPassedTestRepository.cs [hunk 8]
```diff
@@ -12,7 +12,7 @@ public interface IPassedTestRepository : IGenericRepository<PassedTest>
...
```
Comment (artemgael96) [inline exalearn-api/ExaLearn.Dal/Interfaces/IPassedTestRepository.cs]: rename to id

**Pull request merged**